css/shop.css

```diff
@@ -0,0 +1,174 @@
+/* Layout */
+.container {
+    display: grid;
+    height: 100vh;
+    grid-template-columns: 22% 1fr;
+    grid-template-rows: auto 1fr auto;
+    grid-template-areas:
+        "nav nav"
+        "filter products"
+        "footer footer";
+}
+
+nav {
+    grid-area: nav;
+}
+
+.filter__container {
+    grid-area: filter;
+    padding: 10%;
+}
+
+footer {
+    grid-area: footer;
+}
+
+/* Filter styling */
+
+.filter__container hr {
+    margin: 0.8rem auto;
+    border: 1px solid #ddd;
+}
+
+.filter__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5rem;
+}
+
+.filter__header .filter__main-title {
+    font-size: 1.35rem;
+    font-weight: 500;
+    color: var(--color-dark);
+    font-family: "Roboto", sans-serif;
+}
+
+.filter__section {
+    font-size: 1rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 0.5rem;
+}
+
+.filter__section fieldset {
+    display: flex;
+    flex-direction: column;
+    border: 0;
+    padding: 1rem 0;
+}
+
+.filter__section .filter__section-title {
+    font-size: 1.15rem;
+    font-weight: 500;
+    color: var(--color-dark);
+}
+
+.filter__section input[type="radio"],
+.filter__section input[type="checkbox"] {
+    margin-right: 0.8rem;
+    cursor: pointer;
+}
+
+/* Filter slider for price */
+
+.slider {
+    appearance: none;
+    background: #d3d3d3;
+    opacity: 0.7;
+    transition: opacity 0.3s;
+    border-radius: 10px;
+    width: 100%;
+}
+
+.slider__field {
+    position: relative;
+    padding: 1rem 0;
+    margin-bottom: 1rem;
+}
+
+.slider__value {
+    position: absolute;
+}
+
+.slider__value--left {
+    left: 0;
+    bottom: -20px;
+}
+
+.slider__value--right {
+    right: 0;
+}
```

Comment:
You have used BEM naming convention for class which is pretty neat. Class names are descriptive and easy to understand. Files sre readable and clean. Great work. Go ahead and merge the branch.